src/Form/Form.php

```diff
@@ -318,11 +318,13 @@ public function strings($defaults = false): array
 	 */
```

Comment:
`Suggestion` as a user of `$form->submit()` may not be aware about `$fields->submit()`:

```suggestion
	 *
	 * @param bool $passthrough If true, values for undefined fields will be submitted
	 * @param bool $force If true, values for fields that cannot be submitted (e.g. disabled or untranslatable fields) will be submitted
	 */
```